### PR TITLE
Pecan PIE doc updates.

### PR DESCRIPTION
**Pull request by amanpatel**
Pecan PIE gene list section updated to match with the updates on the
website. "Germline reviewable" list is referred to as "Cancer
predispostion" according to JZ's wishes.

### CODE REVIEW

File: docs/guides/tools/pecan-pie.md [hunk 1]
```diff
@@ -69,18 +69,22 @@ uncompressed or compressed with `bgzip`.
 
 ## Advanced options
 
-The "Advanced option" panel lets you customize the behavior of the
+The "Advanced option" panel lets you customize the behavior of th
```
Comment (zaeleus) [inline docs/guides/tools/pecan-pie.md]:
```suggestion
The "Advanced option" panel lets you customize the behavior of the
```

File: docs/guides/tools/pecan-pie.md [hunk 2]
```diff
@@ -69,18 +69,22 @@ uncompressed or compressed with `bgzip`.
...
-* Curated germline genes only: if checked, this filters your variants
-  to disease-related genes only; this option will be turned on
-  automatically if your uploaded file is 1 megabyte or larger and you
-  are not using a custom gene list. See the [frequently asked questions](#frequently-asked-questions) for more information. 
-  This option reduces the variant processing burden on PIE by removing variants
-  that will not be assigned a medal in any case because they are not
-  on the reviewable gene list.
-* Specify custom gene list: here you can override the disease-related
-  gene list used by Pecan PIE with your own list.
+* Gene list: Pick a gene list from the pulldown, this filters your
```
Comment (zaeleus) [inline docs/guides/tools/pecan-pie.md]:
```suggestion
* Gene list: Pick a gene list from the pulldown. This filters your
```

File: docs/guides/tools/pecan-pie.md [hunk 3]
```diff
@@ -69,18 +69,22 @@ uncompressed or compressed with `bgzip`.
...
+  variants to genes in the specified list. This option is required and turned
+  on automatically if your uploaded file is 2 megabyte or larger. See the
```
Comment (zaeleus) [inline docs/guides/tools/pecan-pie.md]:
```suggestion
  on automatically if your uploaded file is 2 megabytes or larger. See the
```

File: docs/guides/tools/pecan-pie.md [hunk 4]
```diff
@@ -69,18 +69,22 @@ uncompressed or compressed with `bgzip`.
...
+  [frequently asked questions](#frequently-asked-questions) for more
+  information. This option reduces the variant processing burden on PIE by
+  removing variants that will not be assigned a medal in any case because
+  they are not on the cancer predisposition gene list. You can review the
+  genes by clicking on the link that will appear just below the pull down
+  titled "See gene list".
+* Custom gene list: Choosing "custom" as your gene list will open a
+  window that will let you paste in a list of genes. Any invalid genes will
+  be dropped from your list automatically. You can separate your genes spaces
```
Comment (zaeleus) [inline docs/guides/tools/pecan-pie.md]:
```suggestion
  be dropped from your list automatically. You can separate your genes by spaces
```

File: docs/guides/tools/pecan-pie.md [hunk 5]
```diff
@@ -371,11 +375,11 @@ order to query data for targeted genes.
 
 **Q: Are there limits on the size of VCF files?**
 
-Uploaded files must not exceed 4 gigabytes. If an uploaded file is
-larger than 1 megabyte, reviewable-gene filtering will be
-automatically enabled unless you are using a custom gene list. This
-reduces the processing burden on the system by removing variants
-outside of targeted genes.
+Uploaded files must not exceed 4 gigabytes. If an uploaded file is larger
+than 2 megabyte, cancer predisoposition gene list filter will be
```
Comment (zaeleus) [inline docs/guides/tools/pecan-pie.md]:
```suggestion
than 2 megabytes, the cancer predisposition gene list filter will be
```

**Comment by amanpatel** [comment]
Michael, thanks for the review & the many fixes.

**Comment by zaeleus** [comment]
@amanpatel The suggestions are not applied automatically. You can either manually make the edits and push a new commit or [use GitHub to commit the changes](https://help.github.com/articles/incorporating-feedback-in-your-pull-request/).

**Comment by mtreadwa** [comment]
I can't tell that Michael's edits have been incorporated, and I only see one commit under this update. Aman can you accept Michael's edits and commit again?

**Comment by claymcleod** [comment]
@mtreadwa Should be good to merge from here.

**Pull request merged**